ticketevolution/seatmaps-client
# Fixed default module export

## Changelog
- removed unused `chunkFileName` option in production webpack config
- fixed default module export by setting `libraryExport` to `default` in production webpack config

## Public Changes
```js
const map = new Tevomaps.default()
```
becomes
```js
const map = new Tevomaps()
```

## Testing Instructions
1. `yarn build`
1. open `./examples/using-lib/index.html` in your favorite web browser
1. you should now see a sample Tevomap 🎊 🌮
